pandas/core/groupby/ops.py

```diff
@@ -1027,7 +1028,10 @@ def agg_series(
         else:
             result = self._aggregate_series_fast(obj, func)
 
-        npvalues = lib.maybe_convert_objects(result, try_float=False)
+        convert_datetime = obj.dtype.kind == "M"
```

Comment:
what breaks if we remove this inference entirely?

Reply:
huh, nothing now.  im pretty sure there was something back when i did this.  will revert

Reply:
updated + green